store/src/java/com/zimbra/cs/account/ldap/LdapProvisioning.java

```diff
@@ -380,7 +380,6 @@ public LdapProvisioning(CacheMode cacheMode) {
 
         Set<String> attrs = Sets.newHashSet(dlAttrs);
         attrs.add(Provisioning.A_objectClass);
-        attrs.remove(Provisioning.A_zimbraMailForwardingAddress);  // the member attr
```

Comment:
this might break existing functionality where mailForwardingAddress attribute is not returned in response.